pkg/apis/autoscaling/register.go

```diff
@@ -62,13 +62,18 @@ const (
 	Concurrency = "concurrency"
 	// CPU is the amount of the requested cpu actually being consumed by the Pod.
 	CPU = "cpu"
+	// Memory is the amount of the requested memory actually being consumed by the Pod.
+	Memory = "memory"
 	// RPS is the requests per second reaching the Pod.
 	RPS = "rps"
 
 	// TargetAnnotationKey is the annotation to specify what metric value the
 	// PodAutoscaler should attempt to maintain. For example,
 	//   autoscaling.knative.dev/metric: cpu
 	//   autoscaling.knative.dev/target: "75"   # target 75% cpu utilization
+	// Or
+	//   autoscaling.knative.dev/metric: memory
+	//   autoscaling.knative.dev/target: "100"   # target 100MiB utilization
```

Comment:
In v2beta2 version of hpa object a typical setup is:
```
  metrics:
  - type: Resource
    resource:
      name: cpu
      target:
        type: Utilization
        averageUtilization: 50
  - type: Resource
    resource:
      name: memory
      target:
        type: AverageValue
        averageValue: 100Mi
```
Memory has a different type than cpu. So technically it is not utilization.

Reply:
sorry for the wrong  comment， now actually using `AverageValue` https://github.com/knative/serving/pull/11668/files#diff-e408734127c500d2019b9bfa1c9700813d4fabba6a2f99766b34bf91f533750eR81

Reply:
Want to adjust the comment to `# target 100MiB memory usage` then, just to avoid that confusion?